packages/create-cloudflare/src/cli.ts

```diff
@@ -19,6 +19,7 @@ import {
 	rectifyPmMismatch,
 } from "helpers/packageManagers";
 import { installWrangler, npmInstall } from "helpers/packages";
+import { validateMacOSVersion } from "../../wrangler/src/utils/validate-macos-version";
```

Comment:
We can't import across package boundaries like this.
If we want to share this how about exporting it from wrangler and then importing as `from "wrangler";`?